operator-common/src/main/java/io/strimzi/operator/PlatformFeaturesAvailability.java

```diff
@@ -205,11 +205,8 @@ public boolean isOpenshift() {
     }
 
     public boolean isNamespaceAndPodSelectorNetworkPolicySupported() {
```

Comment:
We should remove this method completely since it is only returning true now ... and also remove all its uses of course. We can do it in separate PR => but in that case we should open a new issue for it and link it here in a comment.

Reply:
Yes, I agree. I didn't removed it completely because it was used too widely in the code base.